Geometry/HcalCommonData/BuildFile.xml

```diff
@@ -1,6 +1,7 @@
 <use   name="DetectorDescription/Base"/>
 <use   name="DetectorDescription/Core"/>
 <use   name="DataFormats/HcalDetId"/>
+<use   name="SimDataFormats/CaloTest"/>
```

Comment:
I shall move TestNumbering to DataFormats in the next round - because it is used in places beyond SIM step. In any case DIGI is still simulation